cmake/FindAPR.cmake

```diff
@@ -38,7 +38,7 @@ FIND_PATH(APR_INCLUDE_DIR apr.h
     /usr/local/apr/include/apr-1
 )
 
-SET(APR_NAMES ${APR_NAMES} libapr-1 apr-1)
```

Comment:
Need to keep libapr-1 as well for older ubuntu versions - can you include that as well please? Otherwise looks good.